test/example/basic_integration_test.test

```diff
@@ -1,9 +0,0 @@
-<launch>
```

Comment:
why did you delete these examples??

Reply:
yeah definitely add them back, i think we want the simple example templates

Reply:
kk